### DISCUSSION TITLE
Topic/Guide/Content suggestions for the Developer Portal

### DISCUSSION THREAD

**Discussion by katomm** [cardano-foundation/developer-portal#161] [General]
Please reply here if you have a topic suggestion for the Developer Portal. (unless the suggestion is worth opening its own issue) - if possible please also name the category where the suggested content is missing.

**Comment by tweakch** [comment]
did I jump the gun here? see  #154

**Reply by katomm** [reply]
Not at all. The purpose of this thread is mainly to capture the opinions of people who are not used to working with GitHub.

**Reply by sebpab** [reply]
Hi Developer Portal link.  I was reading your great Koios Team interview.  I think you should make an interview about Dandelion API.  By reference, Dandelion is a service similar to Koios.  And also it's one of the most popular, Open Source, descentralized, free, in the Cardano ecosystem.  The team behind Dandelion is Gimbalabs, the infraestructure and teaching center responsable for successfull Cardano startups like Loxe, jpg.store, Rarety, Martify, and others.  The Gimbalabs co-founder James Dunseith and the Gimbalas co-founder and creator of Dandelion, Roberto Morano, will be open to talk with you.  James email: jdunseith@gimbalabs.io , Roberto's email: rcmova@gmail.com , James Discord: jamesdunseith#3315 .  Thanks.

**Reply by katomm** [reply]
Thank you @sebpab, great suggestion. I love the people from Gimbalabs. For anyone interested in doing a developer spotlight interview, please use this form: http://cardanofoundation.org/forms/devspotlight

**Comment by sebpab** [comment]
Thanks Tommy, glad to receive your email.

Fantastic! Thanks for the support.

**Comment by btbf** [comment]
[Ethereum.org developers pages](https://ethereum.org/en/developers/docs/) supports multiple languages. 
We feel that now is the time to support multiple languages.

Are there any concerns in making this happen?

**Reply by rphair** [reply]
From the level of material support that the Cardano Foundation has shown lately for the Developer Portal and the CIP process, I would guess the only way this would be possible would be through a sizeable Project Catalyst bid in Fund 11 with a proposal scope to compensate existing CF Cardano Ambassadors qualified to submit technical translations in each of the target languages. 🤔

**Comment by Quetzacuatl** [comment]
Hi Sir, 

As a "not so seasoned" programmer - tried following : https://developers.cardano.org/docs/get-started/installing-cardano-node#linux

suggestions : 
- ghcup install ghc 8.10.7 --> who updates this in case newer versions are suggested ?  (same for cabal version)
- spent 4 hours being investigating why "cabal build all" gives this error ..
- Not really stimulating if following a 1st install page takes 4 hours and still not working  :) (probably my own dumb fault)
![Schermafbeelding 2024-06-11 220356](https://github.com/cardano-foundation/developer-portal/assets/50581123/f6669475-b12f-49e7-aa04-3e05597009ec)


Error: cabal: Could not resolve dependencies:
[__0] next goal: cardano-git-rev (user goal)
[__0] rejecting: cardano-git-rev-0.2.2.0, cardano-git-rev-0.2.1.0,
cardano-git-rev-0.2.0.0 (constraint from user target requires ==0.1.3.0)
[__0] trying: cardano-git-rev-0.1.3.0
[__1] trying: base-4.20.0.0/installed-4014 (dependency of cardano-git-rev)
[__2] next goal: cardano-node (user goal)
[__2] rejecting: cardano-node-8.10.1 (constraint from user target requires
==8.9.3)
[__2] trying: cardano-node-8.9.3
[__3] trying: yaml-0.11.11.2 (dependency of cardano-node)
[__4] trying: unordered-containers-0.2.20 (dependency of yaml)
[__5] next goal: hashable (dependency of unordered-containers)
[__5] rejecting: hashable-1.4.3.0 (conflict: base==4.20.0.0/installed-4014,
hashable => base>=4.10.1.0 && <4.20)
[__5] skipping: hashable-1.4.2.0, hashable-1.4.1.0, hashable-1.4.0.2,
hashable-1.4.0.1, hashable-1.4.0.0, hashable-1.3.5.0, hashable-1.3.4.1,
hashable-1.3.4.0, hashable-1.3.3.0, hashable-1.3.2.0, hashable-1.3.1.0,
hashable-1.3.0.0, hashable-1.2.7.0, hashable-1.2.6.1, hashable-1.2.6.0,
hashable-1.2.5.0, hashable-1.2.4.0, hashable-1.2.3.3, hashable-1.2.3.2,
hashable-1.2.3.1, hashable-1.2.3.0, hashable-1.2.2.0, hashable-1.2.1.0,
hashable-1.2.0.10, hashable-1.2.0.9, hashable-1.2.0.8, hashable-1.2.0.7,
hashable-1.2.0.6, hashable-1.2.0.5, hashable-1.2.0.4, hashable-1.2.0.3,
hashable-1.2.0.2, hashable-1.2.0.1, hashable-1.2.0.0, hashable-1.1.2.5,
hashable-1.1.2.4, hashable-1.1.2.3 (has the same characteristics that caused
the previous version to fail: excludes 'base' version 4.20.0.0)
[__5] rejecting: hashable-1.1.2.2 (conflict: unordered-containers =>
hashable>=1.4 && <1.5)
[__5] skipping: hashable-1.1.2.1, hashable-1.1.2.0, hashable-1.1.1.0,
hashable-1.1.0.0, hashable-1.0.1.1, hashable-1.0.1.0, hashable-1.0.0 (has the
same characteristics that caused the previous version to fail: excluded by
constraint '>=1.4 && <1.5' from 'unordered-containers')
[__5] fail (backjumping, conflict set: base, hashable, unordered-containers)
After searching the rest of the dependency tree exhaustively, these were the
goals I've had most trouble fulfilling: base, hashable, unordered-containers,
yaml, yaml:nolib, cardano-node, cardano-git-rev
Try running with --minimize-conflict-set to improve the error message.


Thanks for the help !
kind regards,
Pieter

**Reply by rphair** [reply]
This isn't the right place to get help for these questions but there is an informed & helpful community with lots of node building experience here: https://t.me/CardanoStakePoolWorkgroup

Note a lot of dependency problems can be resolved by running `cabal update` (I always run it 3 times, mainly for metaphysical reasons)

**Comment by MurilloBrandao** [comment]
I'm not sure if the Cardano platform is the ideal place to launch an innovative proposal in the realm of collaborative human interactions. This has been the focus of my attention for many years, and it seems to me that in current virtual collaborative networks, the lack of adequate tools leads to a collective silence on the behavioral issues that stimulate, and hinder, collaboration. Something so important and yet so silenced. Several authors worry about the isolation in bubbles that can significantly affect the quality of our interactions in cyberspace. My invitation is to facilitate collaboration between people. I suggest a topic for this, titled "Humanities."

I have developed a method and a tool to encourage healthy conversations on sensitive topics. Conversations necessary, for example, to align a team before embarking on their sales strategy or any other venture. To facilitate these conversations, we created a specific video glossary characterizing twenty-eight "virtues of collaboration." The method explores the qualities of the team, required by a specific team in their context.
Please tell me if we should advance in such a direction. I should also say that I am illiterate in GitHub and in others developer tools.
Thank you,
Murillo Brandao.

**Reply by sebpab** [reply]
@MurilloBrandao where can I read more about the content you mention (humanities, your tooling etc)? Thanks.

**Reply by MurilloBrandao** [reply]
[Human Values in Computing - Is the Objectification of the Subjectivity, a Prior Step  - cópia.pdf](https://github.com/user-attachments/files/22541094/Human.Values.in.Computing.-.Is.the.Objectification.of.the.Subjectivity.a.Prior.Step.-.copia.pdf)

**Reply by MurilloBrandao** [reply]
In this paper we explore the possibilities of an object (possible understood to machines) to facilitate a human machine interactions, based in the same video glossary of collaborative virtues. To treat as an object the collective of subjectives perspectives.

**Reply by MurilloBrandao** [reply]
If somebody consider to develop the mater human collaboration, just let me known it.

**Reply by MurilloBrandao** [reply]
I will collaborate with him.
